packages/admin-ui/src/actions/application.js

```diff
@@ -177,8 +178,38 @@ function* loadActions() {
   }
 }
 
-export default function* watchRequestedMenu() {
+export const COMPONENTS_REQUESTED = 'COMPONENTS_REQUESTED';
+export const requestComponents = () => ({
+  type: COMPONENTS_REQUESTED,
+  payload: {},
+});
+
+export const COMPONENTS_LOADED = 'COMPONENTS_LOADED';
+
+function* loadComponents() {
```

Comment:
How about calling this `loadComponentInfo` or `loadComponentList` since we are not actually loading the components but the information about components that exist?

Reply:
ON top of the changes of @mattgrill I also renamed the constant / other methods.